login-workflow/src/components/SetPassword/SetPassword.tsx

```diff
@@ -84,13 +86,15 @@ export const SetPassword: React.FC<React.PropsWithChildren<SetPasswordProps>> =
                 inputRef={passwordRef}
                 label={newPasswordLabel}
                 value={passwordInput}
+                error={shouldValidatePassword && !isValidPassword()}
+                sx={TextFieldStyles(theme)}
+                {...passwordTextFieldProps}
```

Comment:
Any particular reason the `passwordTextFieldProps` is spreaded here as opposed to spreaded towards the end of this props list? I'd like to hear your reasoning.

Reply:
Same for the other spreads you added in this pull request

Reply:
To override the defaults which we set in the base screen components, spreaded the `TextFieldProps` towards the end of the props list.

Reply:
I do not understand. I see that you still have `onChange`, `onKeyUp` and `onBlur` under the `{...passwordTextFieldProps}`, so this is not the end of the props list of `PasswordTextField`?

Reply:
Just had a discussion with Joe in Office hours regarding this, so we need to merge events with `TextFieldProps` like we did for `WorkFlowActions`.